.circleci/config.yml

```diff
@@ -321,18 +320,6 @@ jobs:
           command: |
             npx lerna run build --scope='@automattic/wpcom-block-editor' -- -- --output-path=$CIRCLE_ARTIFACTS/wpcom-block-editor
       - store-artifacts-and-test-results
-
-  build-full-site-editing:
-    <<: *defaults
-    parallelism: 1
-    steps:
-      - prepare
-      - run:
-          name: Build the Full Site Editing plugin and theme
-          command: |
-            npx lerna run build --scope='@automattic/full-site-editing'
-            tar -czvf $CIRCLE_ARTIFACTS/full-site-editing/build-archive.tar.gz -C apps/full-site-editing/full-site-editing-plugin .
-      - store-artifacts-and-test-results
   test-full-site-editing:
```

Comment:
@noahtallen do we still need this?

Reply:
we do need test FSE, it runs our JS unit tests. Not sure if build FSE is required for that? If so, then we could just add a build step to the tests command

Reply:
We'll know once the jobs are complete. 😄

Reply:
> we do need test FSE, it runs our JS unit tests.

Oh ok, I wasn't sure if we had actions equivalent of that at this point.

Reply:
nope :)